src/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>LeaderBoard</title>
+</head>
+
+<body>
+    <section class="container">
+        <div id="title">
+            <h1>Leaderboard</h1>
+        </div>
+        <div id="wrapper">
+            <div class="score-board">
+                
+                <div class="refreshbtn">
+                    <h2>Recent scores</h2>
+                    <button>Refresh</button>
+                </div>
+
+                <ul>
+                    <li>Name: 100</li>
+                    <li>Name: 20</li>
+                    <li>Name: 50</li>
+                    <li>Name: 78</li>
+                    <li>Name: 125</li>
+                    <li>Name: 77</li>
+                    <li>Name: 42</li>
+                </ul>
+            </div>
+            <div id="score_input">
+                <h2>Add your score</h2>
+                <form action="" method="GET">
+                    <input type="text" name="name" id="name" required placeholder="Your name" />
+                    <input type="text" name="score" id="score" required placeholder="Your score" />
+                    <button type="submit">Submit</button>
+                </form>
+            </div>
+        </div>
```

Comment:
- [ ] Please align the `Submit` button to the right instead of the center as on the wireframe design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/215796443-27d0d0ef-6569-45cd-ab56-cdcdf50e8cf7.png)

### Wireframe design

![image](https://user-images.githubusercontent.com/83803414/215796208-3dee6027-69e0-4bb3-8c40-5e6b11b95445.png)